packs/spell-effects/spell-effect-blink-charge.json

```diff
@@ -0,0 +1,81 @@
+{
+    "_id": "lUL4nvIVYqFOwxAZ",
+    "img": "icons/magic/sonic/projectile-shock-wave-blue.webp",
+    "name": "Spell Effect: Blink Charge",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Blink Charge]</p>\n<p>Your Strike deals extra force damage, depending on the result of your Strike.</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 0
+        },
+        "level": {
+            "value": 5
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Secrets of Magic"
+        },
+        "rules": [
+            {
+                "choices": {
+                    "ownedItems": true,
+                    "predicate": [
+                        "item:equipped"
+                    ],
+                    "types": [
+                        "weapon"
+                    ]
+                },
+                "flag": "weapon",
+                "key": "ChoiceSet"
+            },
+            {
+                "damageType": "force",
+                "diceNumber": "ceil((@item.level - 2)/2)",
+                "dieSize": "d8",
+                "key": "DamageDice",
+                "selector": "{item|flags.pf2e.rulesSelections.weapon}-damage"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.blinkCharge",
```

Comment:
Is this needed? I thought `@item.level` was accessible to the Note?

Reply:
Yes, otherwise it'd be using the level of the item the Note is attached to.